demos/coreMQTT/CMakeLists.txt

```diff
@@ -1,14 +1,15 @@
-# AFR coreMQTT demos
+# AFR coreMQTT demo
```

Comment:
One demo target is used to represent multiple MQTT demos, thus, this should still use `demos`
```suggestion
# FreeRTOS coreMQTT demos
```